Add Exemplars, Imitating the Standards

KALU-KELECHI-GABRIEL opened the pull request:
I have added example papers (and artifact links) for Empirical measurements, Human studies, Experiments and quasi Experiments in the open-science.md file

KALU-KELECHI-GABRIEL committed:
Update OpenScience.md

I have added example papers (and artifact links) for Empirical measurements, Human studies, Experiments and quasi Experiments

davisjam commented on docs/supplements/OpenScience.md:
```diff
@@ -59,3 +60,25 @@ Daniel Méndez, Daniel Graziotin, Stefan Wagner, and Heidi Seibold. 2019. Open s
+
+#### Method: Experiments (System Evaluations, etc) and Quasi-Experiments
+
+Chen, M., Tan, T., Pan, M., & Li, Y. (2025, March). PacDroid: A Pointer-Analysis-Centric Framework for Security Vulnerabilities in Android Apps. In 2025 IEEE/ACM 47th International Conference on Software Engineering (ICSE) (pp. 744-744). IEEE Computer Society. (Artifact). Zenodo. [https://doi.org/10.5281/zenodo.14863334](https://doi.org/10.5281/zenodo.14863334)
```
Let's typeset as a bullet list, not just raw text

davisjam replied:
Citation style is inconsistent between the papers, I think. This first one says "(2025, March)" and the next one doesn't do that.

davisjam commented on docs/supplements/OpenScience.md:
```diff
@@ -59,3 +60,25 @@ Daniel Méndez, Daniel Graziotin, Stefan Wagner, and Heidi Seibold. 2019. Open s
+## Examplar Artifacts
+Below are example artifacts for different types of software engineering research to aid in preparing reusable and functional research artifacts.
+
+#### Method: Experiments (System Evaluations, etc) and Quasi-Experiments
```
The headings here should match the standards headings.

https://www2.sigsoft.org/EmpiricalStandards/docs/standards

davisjam commented on docs/supplements/OpenScience.md:
```diff
@@ -59,3 +60,25 @@ Daniel Méndez, Daniel Graziotin, Stefan Wagner, and Heidi Seibold. 2019. Open s
+
+T. R. Schorlemmer, K. G. Kalu, L. Chigges, et al., “Signing in four public software package registries: Quantity, quality, and influencing factors,” in 2024 IEEE Symposium on Security and Privacy (SP), Los Alamitos, CA, USA: IEEE Computer Society, May 2024. (Artifact) Github [https://github.com/PurdueDualityLab/signature-adoption](https://github.com/PurdueDualityLab/signature-adoption)
+ 
+#### Method: Empirical Measurements (Non-human studies)
```
The headings here should match the standards headings.

https://www2.sigsoft.org/EmpiricalStandards/docs/standards

davisjam commented on docs/supplements/OpenScience.md:
```diff
@@ -59,3 +60,25 @@ Daniel Méndez, Daniel Graziotin, Stefan Wagner, and Heidi Seibold. 2019. Open s
+Xinchen Wang, Ruida Hu, Cuiyun Gao, Xin-Cheng Wen, Yujia Chen, and Qing Liao. 2024. ReposVul: A Repository-Level High-Quality Vulnerability Dataset. In Proceedings of the 2024 IEEE/ACM 46th International Conference on Software Engineering: Companion Proceedings (ICSE-Companion '24). Association for Computing Machinery, New York, NY, USA, 472–483. (Paper) https://doi.org/10.1145/3639478.3647634. ( ...
+
+ 
+#### Method: Human Studies (Interviews, Surveys, etc)
```
The headings here should match the standards headings.

https://www2.sigsoft.org/EmpiricalStandards/docs/standards

davisjam reviewed (changes requested):
Fix

KALU-KELECHI-GABRIEL committed:
Fix  Examplar typesetting

I have made edits to fix the headings (to match the Standards) , to make the citations consistent, and to list the exemplars as bullet lists.

KALU-KELECHI-GABRIEL commented:
I have fixed the issues (with the new commit) in line withe feedbacks - Ready for review

davisjam committed:
Merge pull request #10 from acmsigsoft/master

get latest

davisjam commented:
@KALU-KELECHI-GABRIEL 

1. We should only reference artifacts associated with papers in flagship venues.
2. I think we can reference at most one of our own group's artifacts or else we look a bit silly. Taylor's artifact is great, so we can use that one. Using any others from our group, I think it's not appropriate. Too self-congratulatory.
3. There are many research methods. What made you decide to pick only these three to illustrate with exemplars?
4. Have you considered looking at winners of "best artifact" awards to select?

KALU-KELECHI-GABRIEL committed:
Add examplars for other methods

I have expanded examples to other methods from flagship venues as required

davisjam commented:
I made some final edits myself and opened the PR. It is https://github.com/acmsigsoft/EmpiricalStandards/pull/168
